packages/xchain-litecoin/src/client.ts

```diff
@@ -194,10 +198,13 @@ class Client implements LitecoinClient, XChainClient {
    * @throws {"Phrase must be provided"} Thrown if phrase has not been set before.
    * @throws {"Address not defined"} Thrown if failed creating account from phrase.
    */
-  getAddress = (): Address => {
-    if (this.phrase) {
+  getAddress = (index = 0): Address => {
+    if (index < 0) {
+      throw new Error('index must be greater than zero')
+    }
+    if (this.phrase !== '') {
```

Comment:
Wouldn't be a guard like `if(this.phrase) {}` (as we had as before) more safety?